pkg/config/config.go

```diff
@@ -116,6 +117,11 @@ const (
 	DefaultIrqBalanceConfigFile = "/etc/sysconfig/irqbalance"
 )
 
+const (
+	// DefaultRdtConfigFile is the default value for RDT config file path
+	DefaultRdtConfigFile = ""
+)
+
```

Comment:
Let's move this const to `internal/config/rdt/rdt.go`.

Reply:
Moved